include/ck/tensor_operation/gpu/element/binary_element_wise_operation.hpp

```diff
@@ -101,6 +101,110 @@ struct Scales
     }
 };
 
+struct Max
```

Comment:
Max, Min and Multiply are already implemented, please rebase

Reply:
https://github.com/ROCm/composable_kernel/blob/381d44aa60501098baef69bc78add2c24adb4ba4/include/ck/tensor_operation/gpu/element/binary_element_wise_operation.hpp#L95

Reply:
Done